runtime/embed/process.go

```diff
@@ -94,6 +97,44 @@ func process(worker *Worker, loggingBackend sqlq.LogBackend) (shutdown func(time
 						result <- panicWrap(func() error { return fn.Process(jobContext, job) })
 					}()
 
+					// we run a polling checking every 3 seconds whether the job have a
+					// cancelling status if so we change it's status to cancelled.
+					go func() {
+						var poll = func(job *sqlq.Job) (r int64, err error) {
+							var tx *sql.Tx
+							var rows int64
```

Comment:
Can you move this `var rows int64` declaration down to where it is used first? on `L110`?

Reply:
Done